_schema.graphql

```diff
@@ -1032,7 +1032,7 @@ type ArtworkContextFair {
     section: String
 
     # Sorts for shows in a fair
-    sort: ShowSort = FEATURED_DESC
+    sort: ShowSort = null
```

Comment:
not sure why these are here

Reply:
This is something to do with schema stitching - it's blocking me on 
https://github.com/artsy/metaphysics/pull/1311 too

Reply:
I'll fix it today and unblock this PR